clang/test/CodeGen/arithmetic-fence-builtin.c

```diff
@@ -32,7 +42,11 @@ int addit(float a, float b) {
 
   v = (a + b);
   // CHECKPP: call{{.*}} float @llvm.arithmetic.fence.f32(float %add{{.*}})
```

Comment:
If we are allowing this in FE, it can reach SPIR-V translator. Is there SPIR-V translator support already?

Reply:
May be @MrSidims  would know that?

Reply:
It's supported under [SPV_INTEL_arithmetic_fence](https://github.com/intel/llvm/pull/3914) extension